scipy/signal/_upfirdn.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
```

Comment:
My editor does it automatically, but I agree it's probably not necessary and should be removed for consistency with the repo.